components/workflow-mgt/org.wso2.carbon.identity.workflow.mgt/src/main/java/org/wso2/carbon/identity/workflow/mgt/dao/WorkflowDAO.java

```diff
@@ -137,6 +137,26 @@ public void removeWorkflow(String workflowId) throws InternalWorkflowException {
         }
     }
 
+    /**
+     * Remove all workflows of a given tenant id.
+     *
+     * @param tenantId Id of the tenant
+     * @throws InternalWorkflowException
+     */
+    public void removeWorkflows(int tenantId) throws InternalWorkflowException {
+
+        try (Connection connection = IdentityDatabaseUtil.getDBConnection(true)) {
+            try (PreparedStatement prepStmt = connection.prepareStatement(SQLConstants
+                    .DELETE_WORKFLOW_BY_TENANT_ID_QUERY)) {
+                prepStmt.setInt(1, tenantId);
+                prepStmt.executeUpdate();
+                IdentityDatabaseUtil.commitTransaction(connection);
+            }
+        } catch (SQLException e) {
+            throw new InternalWorkflowException(errorMessage, e);
```

Comment:
Where the errorMessage is defined?

Reply:
Fixed in #3312

Reply:
It is defined in the beginning of the file, [here](https://github.com/sumedhe/carbon-identity-framework/blob/12fbf8f7c88c6d2603327181d5543b0a1329cdd4/components/workflow-mgt/org.wso2.carbon.identity.workflow.mgt/src/main/java/org/wso2/carbon/identity/workflow/mgt/dao/WorkflowDAO.java#L41).